### ISSUE TITLE
Torch 1.1.0 Support?

### ISSUE THREAD

**Issue by timonbimon** [kmkurn/pytorch-crf#39]
Hey :) 
Are there any plans to support PyTorch 1.1 any time soon?

**Comment by kmkurn** [comment]
Hi, I'd love to support PyTorch 1.1 but unfortunately my schedule has been quite full for some time now so I won't be able to do it anytime soon :(

**Comment by timonbimon** [comment]
Ok thanks for the reply and thanks for contributing this in the first place :)

**Comment by kmkurn** [comment]
Hi, I just realised that I've put PyTorch 1.1 as a version to test against in travis. Could you check if it indeed works for PyTorch 1.1?

**Comment by nccuSimonLee** [comment]
> Hi, I just realised that I've put PyTorch 1.1 as a version to test against in travis. Could you check if it indeed works for PyTorch 1.1?

Hi! Thanks for your awesome work. It indeed works for PyTorch 1.1.
When I used default initial value of CRF, the performance didn't improve, 
and then I derived transition matrix from training set, making it be the initial value of CRF,
the validation loss almost reduced by half.
I am curious about this weird results. Do you have any idea ?
Thanks again!

**Comment by kmkurn** [comment]
Great! I can close this issue now.

Wow that's impressive. May I know how you exactly derived the transition matrix from the training set?

**Comment by nccuSimonLee** [comment]
I just maintained a matrix (number of tags x number of tags), and went
through the tags in training set.
It's to count the frequency of transition from tag[i] to tag[j].
Finally, each row of the matrix would be normalized by sum of each them.

Kemal Kurniawan <notifications@github.com> 於 2019年8月28日 週三 上午4:46寫道：

**Comment by kmkurn** [comment]
I see. I'm not sure why that works nor why that shouldn't work. But thanks for sharing. Hopefully that helps others too.
